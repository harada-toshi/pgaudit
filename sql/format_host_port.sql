\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! echo "listen_addresses = 'localhost'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log
\! cp sql/format_host_port/format_host_port.conf /tmp/config.conf
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! sleep 1
\! rm /tmp/config.conf
\! sh sql/format_host_port/format_host_port.sh

