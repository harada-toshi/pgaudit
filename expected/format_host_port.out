	        format = 'AUDIT:(A) %class,%remote_host,%remote_port,%connection_message'
	        format = 'AUDIT:(B) %class,%h,%remote_port,%connection_message'
LOG:  AUDIT:(A) CONNECT,[local],,received
LOG:  AUDIT:(B) CONNECT,[local],,received
LOG:  AUDIT:(A) CONNECT,[local],,authorized
LOG:  AUDIT:(B) CONNECT,[local],,authorized
LOG:  AUDIT:(A) CONNECT,[local],,disconnected
LOG:  AUDIT:(B) CONNECT,[local],,disconnected
LOG:  AUDIT:(A) CONNECT,[local],,received
LOG:  AUDIT:(B) CONNECT,[local],,received
LOG:  AUDIT:(A) CONNECT,[local],,authorized
LOG:  AUDIT:(B) CONNECT,[local],,authorized
LOG:  AUDIT:(A) CONNECT,[local],,disconnected
LOG:  AUDIT:(B) CONNECT,[local],,disconnected
